torch/optim/optimizer.py

```diff
@@ -789,6 +789,7 @@ def zero_grad(self, set_to_none: bool = True) -> None:
                 (in one case it does the step with a gradient of 0 and in the other it skips
                 the step altogether).
         """
+        from torch._dynamo.utils import get_static_address_type
```

Comment:
`_dynamo` is a lazy import no? This will actually trigger the lazy import for users that don't use dynamo. that sounds bad?

Reply:
Good catch, I think I'll move this to torch._utils instead